src/main/charts/confluence/values.yaml

```diff
@@ -331,3 +335,12 @@ additionalFiles: []
 #    type: secret
 #    key: keystore.jks
 #    mountPath: /var/ssl
+
+# -- Additional host aliases for each pod, equivalent to adding them to the /etc/hosts file. See
+# https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/ for more
+# information.
+additionalHosts: []
```

Comment:
This is needed for Confluence to be able to contact Synchrony. In the local execution the ingress lives in `.local` Bonjour address. When Confluence pod needs to contact synchrony, it's going to do that using the Ingress address (`.local` hostname). Hence, we must be sure that Ingress address will not be resolved to loopback (127.0.0.1) and will instead be resolved to the IP address of the ingress Service. Otherwise Connie pod will try to send HTTP calls to itself when it needs to contact synchrony and it will fail. 
This `additionalHosts` annotation does exactly what we need - maven determines the LoadBalancer address of the work laptop and passes it to Kube. Hence, pods know how to resolve the Ingress hostname correctly.
This whole situation seems to be a problem only when ingress lives on local address.

Reply:
Is that something that will be useful to have in documentation?

Reply:
It's already in the [internal documentation](https://hello.atlassian.net/wiki/spaces/DCD/pages/1112917002/How%2Bto%2Binstall%2BConfluence%2Bon%2Blocal%2BKubernetes%2Bcluster#Provision-the-cluster) for the local execution. In general, I don't think it's useful in the external docs, because we don't give people configuration for localhost. If we add this support, then we definitely will look at the internal docs when documenting it for customers.